src/cli-client/internals/cli/cli_upload.go

```diff
@@ -0,0 +1,108 @@
+package cli
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/canonical/oci-factory/cli-client/internals/client"
+	"github.com/canonical/oci-factory/cli-client/internals/logger"
+	"github.com/canonical/oci-factory/cli-client/internals/trigger"
+)
+
+type UploadRelease struct {
+	Track     string
+	Risks     []string
+	EndOfLife string
+}
+
+type CmdUpload struct {
+	UploadRelease []string `long:"release" description:"Release images to container registries.\nSyntax: --release tracks=<release track>,risks=<risk1>[,<risk2>...],eol=yyyy-mm-dd"`
+}
+
+var cmdUpload CmdUpload
+var releases = make([]UploadRelease, 0)
+
+func init() {
+	// fmt.Println("DEBUG Adding upload to parser commands")
```

Comment:
These statements should not be part of the PR, either remove them or use `debugf` or something similar if you want to log something for debugging purposes.

Reply:
Makes sense.